core/templates/filters/convert-unicode-with-params-to-html.filter.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('HTML to text', function() {
         var fn = function() {
           return $filter('convertUnicodeWithParamsToHtml')(s);
         };
-        expect(fn).toThrow();
+        expect(fn).toThrowError();
```

Comment:
No message? Add messages where possible, to make the assertion stricter.

Reply:
Added the message